saleor/graphql/menu/mutations.py

```diff
@@ -196,6 +198,103 @@ def user_is_allowed(cls, user, input):
         return user.has_perm('menu.manage_menus')
 
 
+_MenuMoveOperation = namedtuple(
+    '_MenuMoveOperation', ('menu_item', 'parent', 'sort_order'))
+
+
+class MenuItemMove(BaseMutation):
+    menu_item = graphene.List(
+        MenuItem, description='Assigned menu to move within.')
+
+    class Arguments:
+        move = graphene.List(
```

Comment:
Maybe we could rename it to `moves`? This is a list and we also iterate over its items later.